services/app-web/src/pages/StateSubmission/ContractDetails/ContractDetails.test.tsx

```diff
@@ -88,8 +88,8 @@ describe('ContractDetails', () => {
 
         // check that there are the errors we expect
         expect(
-            screen.queryByText('You must select at least one item')
-        ).toBeInTheDocument()
+            screen.queryAllByText('You must select at least one item')
```

Comment:
I can also see us having separate expectations for the field error message and the one in the summary above. WDYT?

Reply:
Yeah either separate assertions or verify the length is 2. I would love to lean more heavily on jest here and be sure we have both inline errors and summaries up. That way we dont need to add much to Cypress.